en/micro-integrator/docs/setup/observability/setting-up-minimum-basic-observability-deployment.md

```diff
@@ -160,217 +145,214 @@ To set up Fluent Bit, follow the procedure below:
-        
-        ingester:
-          lifecycler:
-            address: 127.0.0.1
-            ring:
-              kvstore:
-                store: inmemory
-              replication_factor: 1
-            final_sleep: 0s
-          chunk_idle_period: 5m
-          chunk_retain_period: 30s
-          max_transfer_retries: 0
-        
-        schema_config:
-          configs:
-            - from: 2018-04-15
-              store: boltdb
-              object_store: filesystem
-              schema: v11
-              index:
-                prefix: index_
-                period: 168h
-        
-        storage_config:
-          boltdb:
-            directory: /tmp/loki/index
-        
-          filesystem:
-            directory: /tmp/loki/chunks
-        
-        limits_config:
-          enforce_metric_name: false
-          reject_old_samples: true
-          reject_old_samples_max_age: 168h
-        
-        chunk_store_config:
-          max_look_back_period: 0s
-        
-        table_manager:
-          retention_deletes_enabled: false
-          retention_period: 0s
+    auth_enabled: false
+    
+    server:
+      http_listen_port: 3100
+    
+    ingester:
+      lifecycler:
+        address: 127.0.0.1
+        ring:
+          kvstore:
+            store: inmemory
+          replication_factor: 1
+        final_sleep: 0s
+      chunk_idle_period: 5m
+      chunk_retain_period: 30s
+      max_transfer_retries: 0
+    
+    schema_config:
+      configs:
+        - from: 2018-04-15
+          store: boltdb
+          object_store: filesystem
+          schema: v11
+          index:
+            prefix: index_
+            period: 168h
+    
+    storage_config:
+      boltdb:
+        directory: /tmp/loki/index
+    
+      filesystem:
+        directory: /tmp/loki/chunks
+    
+    limits_config:
+      enforce_metric_name: false
+      reject_old_samples: true
+      reject_old_samples_max_age: 168h
+    
+    chunk_store_config:
+      max_look_back_period: 0s
+    
+    table_manager:
+      retention_deletes_enabled: false
+      retention_period: 0s
     ```
    
- 3. Unzip the file you downloaded in step 1. The directory that is created as a result is referred to as `<GrafanaLoki_Home>` from here onward.
+ 3. Unzip the file you downloaded in step 1. The directory that is created as a result is referred to as `<GrafanaLoki_Home>` from hereon.
  
- 4. Open a new terminal window and navigate to the `<GrafanaLoki_Home>`. Then issue the following command.
+ 4. Open a new terminal and navigate to `<GrafanaLoki_Home>`. 
+ 5. Execute the following command:
  
     `./loki-darwin-amd64 -config.file=./loki-local-config.yaml`
-    
 
 ### Configuring Grafana to visualize logs
 
-In order to configure Grafana to display logs, you need to add Loki as a data source in Grafana. To do this, follow the procedure below:
+In order to configure Grafana to display logs, you need to add Loki as a datasource in Grafana. 
```

Comment:
It should be "data source" as per Jackie's guidelines. Also, Grafana uses "data source".

Reply:
Since we have used datasources in all instances of our docs, will investigate this further and sent another PR>